packages/react-composites/tests/browser/call/hermetic/VideoGallery.test.ts

```diff
@@ -110,14 +112,26 @@ test.describe('VideoGallery tests', async () => {
     await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
 
     // pin remote video tiles up to the max allowed in the call composite
-    for (let i = 0; i < 4; i++) {
-      // click the last 'more options' button on the page which is presumably on an unpinned remote video tile
-      await pageClick(page, dataUiId('video-tile-more-options-button') + ' >> nth=-1');
+    for (let i = 1; i < 5; i++) {
+      await waitForSelector(page, dataUiId(IDS.videoTile) + ` >> nth=${i}`);
+      await page.click(dataUiId(IDS.videoTile) + ` >> nth=${i}`);
+      await waitForSelector(page, dataUiId(IDS.videoTileMoreOptionsButton));
+      // click the next 'more options' button on the page which is presumably on an unpinned remote video tile
+      await screenshotOnFailure(
+        page,
+        async () => await page.click(dataUiId(IDS.videoTileMoreOptionsButton), { timeout: perStepLocalTimeout() })
+      );
       // click pin menu button in contextual menu
       await pageClick(page, dataUiId('video-tile-pin-participant-button'));
+      await page.mouse.move(-1, -1);
     }
-    // click the last more button on the page which is presumably on an unpinned remote video tile
-    await pageClick(page, dataUiId('video-tile-more-options-button') + ' >> nth=-1');
+    // hover the last remote video tile on the page(skip local tile) which is presumably on an unpinned remote video tile
+    await page.hover(dataUiId(IDS.videoTile) + ' >> nth=-2');
```

Comment:
Change the comment and code here to select the 5th video tile. Selecting the 2nd to last tile might be confusing and contradicts the comment.

Reply:
done